ui/src/app/views/app/component.html

```diff
@@ -40,8 +40,8 @@
   <clr-tabs>
     <clr-tab>
       <button clrTabLink routerLink="/" routerLinkActive="active">My Workspaces</button>
-      <button *ngIf="hasAdminPermissions" clrTabLink routerLink="/todo"
-          routerLinkActive="active">Admin</button>
+      <button *ngIf="hasReviewResearchPurpose" clrTabLink routerLink="/review"
```

Comment:
So, I know this hides the button if I don't have access, but if I hard route to `/review`, (type in the URL bar), can I get there/will it show up for me?

Reply:
The page trimmings will display, but the XHR to fetch data will fail with a 401.

Reply:
Sounds good!